resources/public/api/conf/1.0/schemas/create-contact-details-response.json

```diff
@@ -1,3 +1,81 @@
 {
-  "TODO": "TODO"
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Addresses",
+  "type": "object",
+  "properties": {
+    "contactDetails": {
+      "type": "array",
+      "description": "The list of the individual's contact telephone numbers.",
+      "items": {
+        "type": "object",
+        "properties": {
+          "code": {
```

Comment:
Unneeded as no OGDs ask for it -> Same wioll need to be removed from create docs.